test/gui/shared/scripts/pageObjects/AccountStatus.py

```diff
@@ -0,0 +1,58 @@
+import names
+import squish
+
+
+class AccountStatus:
+    ACCOUNT_BUTTON = {
+        "container": names.settings_stack_QStackedWidget,
+        "name": "_accountToolbox",
+        "type": "QToolButton",
+        "visible": 1,
+    }
+    ACCOUNT_MENU = {
+        "type": "QMenu",
+        "unnamed": 1,
+        "visible": 1,
+        "window": names.settings_OCC_SettingsDialog,
+    }
+    SIGNED_OUT_TEXT_BAR = {
+        "container": names.settings_stack_QStackedWidget,
+        "name": "connectLabel",
+        "type": "QLabel",
+        "visible": 1,
+    }
+
+    REMOVE_CONNECTION_BUTTON = {
+        "container": names.settings_stack_QStackedWidget,
+        "text": "Remove connection",
+        "type": "QPushButton",
+        "unnamed": 1,
+        "visible": 1,
+    }
+
+    def loginLogoutAction(self, action):
+        squish.sendEvent(
+            "QMouseEvent",
+            squish.waitForObject(self.ACCOUNT_BUTTON),
+            squish.QEvent.MouseButtonPress,
+            0,
+            0,
+            squish.Qt.LeftButton,
+            0,
+            0,
+        )
+        squish.activateItem(squish.waitForObjectItem(self.ACCOUNT_MENU, action))
+
+    def removeUser(self):
+        squish.sendEvent(
+            "QMouseEvent",
+            squish.waitForObject(self.ACCOUNT_BUTTON),
+            squish.QEvent.MouseButtonPress,
+            0,
+            0,
+            squish.Qt.LeftButton,
+            0,
+            0,
+        )
+        squish.activateItem(squish.waitForObjectItem(names.settings_QMenu, "Remove"))
```

Comment:
instead of this, you can call loginLogoutAction("Remove"). And it would be nice if you rename the function `loginLogoutAction`, it sounds very specific. May be `accountAction` or something like that. 🤔

Reply:
Yes, that can be done but we still need to have `removeUser` method as it does more then just clicking on "Remove". I have refactored this method and made it more DRY.